spec.html

```diff
@@ -37,6 +37,139 @@ <h1>Introduction</h1>
+        <li>has a `name` property whose value is the String value `"<var>AggregateError</var>"`.</li>
+        <li>has the following properties:</li>
+      </ul>
+
+      <emu-clause id="sec-aggregateerror.prototype">
+        <h1>AggregateError.prototype</h1>
+        <p>The initial value of <code><var>AggregateError</var>.prototype</code> is a _AggregateError_ prototype object (<emu-xref href="#sec-properties-of-the-aggregateerror-prototype-objects"></emu-xref>). Each _AggregateError_ constructor has a distinct prototype object.</p>
+        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregateerror-prototype-objects">
+      <h1>Properties of the _AggregateError_ Prototype Objects</h1>
+      <p>Each _AggregateError_ prototype object:</p>
+      <ul>
+        <li>is an ordinary object.</li>
+        <li>is not an Error instance and does not have an [[ErrorData]] internal slot.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+      </ul>
+
+      <emu-clause id="sec-aggregateerror.prototype.constructor">
+        <h1>_AggregateError_.prototype.constructor</h1>
+        <p>The initial value of the `constructor` property of the prototype for a given _AggregateError_ constructor is the corresponding intrinsic object %_AggregateError_% (<emu-xref href="#sec-aggregateerror-constructors"></emu-xref>).</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregateerror.prototype.message">
+        <h1>_AggregateError_.prototype.message</h1>
+        <p>The initial value of the `message` property of the prototype for a given _AggregateError_ constructor is the empty String.</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregateerror.prototype.name">
+        <h1>_AggregateError_.prototype.name</h1>
+        <p>The initial value of the `name` property of the prototype for a given _AggregateError_ constructor is the String value consisting of the name of the constructor (the name used instead of _AggregateError_).</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregateerror.prototype.errors">
+        <h1>_AggregateError_.prototype.errors</h1>
+        <p>The initial value of the `errors` property of the prototype for a given _AggregateError_ constructor is an empty Array.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-aggregateerror-instances">
+      <h1>Properties of _AggregateError_ Instances</h1>
+      <p>_AggregateError_ instances are ordinary objects that inherit properties from their _AggregateError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error or _AggregateError_ instances.</p>
```

Comment:
```suggestion
      <p>_AggregateError_ instances are ordinary objects that inherit properties from their _AggregateError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error, _NativeError_, or _AggregateError_ instances.</p>
```

https://tc39.es/ecma262/#sec-properties-of-nativeerror-instances should be updated accordingly.